docs/flows/flows.puml

```diff
@@ -130,7 +131,50 @@ alt#Gold #DarkSeaGreen trace
   bitdaoadmin <-- bitdaogovernance: TxHash
 end
 
+newpage
+
+autonumber "Insurance:[00]"
+
+== Insurance ==
+
+alice -> insurancecontract: create insurance ticket
+
+
+alice -> insurancecontract: pay for insurance daily / monthly
+
+
+alice -> insurancecontract: compensate
+
+community -> community: delegate votes
+
+alice -> snapshot: submit proposal
+guarantor -> snapshot: show proposals
+guarantor <-- snapshot: proposals
+guarantor -> snapshot: vote
+
+bitdaoadmin -> snapshot: show vote result
+bitdaoadmin <-- snapshot: pass/fail
```

Comment:
suggestion: drop `pass/fail` -> `pass`

As you're showing the passing flow, there's no need to mention the other outcome (unless you're wanting to cover it explicitly).